llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/vec-args.ll

```diff
@@ -407,16 +407,16 @@ entry:
 define void @test_args_nxv64i1(<vscale x 64 x i1> %a) {
   ; RV32-LABEL: name: test_args_nxv64i1
   ; RV32: bb.1.entry:
-  ; RV32-NEXT:   liveins: $v8
+  ; RV32-NEXT:   liveins: $v0
```

Comment:
@michaelmaitland This looks like an existing but in GlobalISel? Masks should be in v0.

Reply:
I will take a look and fix this problem